scripts/index.js

```diff
@@ -1 +1,17 @@
-console.log(`There are ${characters.length} characters in the array.`);
\ No newline at end of file
+console.log(`There are ${characters.length} characters in the array.`);
+
+// console.log(Object.keys(characters));
+//split characters into arrays
+
+function beginsWithLetter(char,letter){
```

Comment:
Nice flexible function. I like that you able to use the same function for both purposes.